src/fun/builtins.bend

```diff
@@ -383,3 +383,60 @@ Bytes/decode_ascii (List/Nil)       = (String/Nil)
 
 String/encode_ascii (String/Cons x xs) = (List/Cons x (String/encode_ascii xs))
 String/encode_ascii (String/Nil)       = (List/Nil)
+
+# Math
+
+# The Pi constant.
+Math/PI = 3.1415926535
+
+# Math/sin(a: f24) -> f24
+# Computes the sine of the given angle in radians.
+hvm Math/sin:
+  ($([<<0x0] a) a)
+
+# Math/cos(a: f24) -> f24
+# Computes the cosine of the given angle in radians.
+Math/cos a = (Math/sin (- (/ Math/PI 2.0) a))
```

Comment:
A slightly faster implementation would be to implement this using the underlying hvm sine with -pi/2 as the second argument.